frontend/src/lib/components/widgets/ChatInput/ChatInput.tsx

```diff
@@ -39,6 +39,7 @@ import {
   StyledSendIconButton,
   StyledSendIconButtonContainer,
 } from "./styled-components"
+import { isInForm } from "src/lib/util/utils"
```

Comment:
St.chat_input[ technically can't be used in forms ](https://docs.streamlit.io/library/api-reference/chat/st.chat_input) but I changed it anyways just because it was a simple fix. However, this involves changing the proto. If we don't want these changes, I can revert them.

Reply:
I think we should keep this out since we kept out all other form-related code from chat input as well. It is likely that we will never support chat input in forms because the triggering value behavior is just inherently incompatible with the form logic.